examples/incompressible-flow/2d-mms/MMS_QUAD/MMS_2D_steady.prm

```diff
@@ -0,0 +1,161 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Lethe Authors
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+# Listing of Parameters
+#----------------------
+
+set dimension = 2
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method = steady
+  set output name = output_MMS_2D_steady_
+  set number mesh adapt = 5
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+
+subsection physical properties
+  subsection fluid 0
+    set kinematic viscosity = 1
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+
+subsection mesh
+  set type               = dealii
+  set grid type          = hyper_cube
+  set grid arguments     = -1 : 1 : true
```

Comment:
```suggestion
  set grid arguments     = -1 : 1 : false
```
If all the boundaries are at zero, then put false here and just have one boundary id. Makes things simpler to understand.